platform/cypress/e2e/collocation/collocation.cy.js

```diff
@@ -1,167 +1,168 @@
-    // Intercept the API request to get the device status summary
-    cy.intercept('GET', '**/summary', {
-      statusCode: 200,
-      body: {
-        data: [
-          {
-            device_name: 'airqo-4001',
-            status: 'running',
-            start_date: '2021-05-20T12:00:00.000Z',
-            end_date: '2021-05-20T12:00:00.000Z',
-          },
-          {
-            device_name: 'airqo-4002',
-            status: 'running',
-            start_date: '2021-05-20T12:00:00.000Z',
-            end_date: '2021-05-20T12:00:00.000Z',
-          },
-        ],
-      },
-    }).as('getDeviceStatusSummary');
-
-    // Visit the collocate page
-    cy.visit('/collocation/collocate');
-
-    // Wait for the API request to finish
-    cy.wait('@getDeviceStatusSummary');
-
-    // Verify that the device status summary table is visible
-    cy.get('[data-testid="collocation-device-status-summary"]').should('be.visible');
-
-    // Use the `eq()` method to click the span element in last row of the table
-    cy.get('table tbody tr:last-of-type span').click({ multiple: true });
-  });
-
-  it('display the correlation charts when the api is successful', () => {
-    // Intercept the API request to get the data
-    cy.intercept('POST', '**/data', {
-      statusCode: 200,
-      body: {
-        devices: ['aq_g520'],
-        startDate: '2023-03-15',
-        endDate: '2023-03-18',
-      },
-    }).as('getCollocationResults');
-
-    // Visit the collocate page
-    cy.visit(
-      '/collocation/reports/monitor_report/aq_g520?devices=aq_g520&startDate=2023-03-15&endDate=2023-03-18',
-    );
-
-    // Wait for the API request to finish
-    cy.wait('@getCollocationResults', { timeout: 5000 });
-
-    // Verify that the chart is visible
-    cy.get('[data-testid="correlation-chart"]').should('be.visible');
-  });
-
-  it('shows the alert when the collocation results api is not successful', () => {
-    // Intercept the API request to get the collocation results
-    cy.intercept('POST', '**/data', {
-      statusCode: 500,
-      body: {
-        devices: ['aq_g520'],
-        startDate: '2023-03-15',
-        endDate: '2023-03-18',
-      },
-    }).as('getCollocationResults');
-
-    // Visit the collocate page
-    cy.visit(
-      '/collocation/reports/monitor_report/aq_g5_87?devices=aq_g5_87&startDate=2023-01-21&endDate=2023-01-24',
-    );
-
-    // Wait for the API request to finish
-    cy.wait('@getCollocationResults', { timeout: 5000 });
-
-    // Verify that the collocation results are visible
-    cy.get('[data-testid="monitor-report-error-toast"]').should('be.visible');
-  });
-});
+// describe('Collocation feature', () => {
+//   it('schedule devices for collocation', () => {
+
+//   // Intercept the API request to get the running devices
+//     cy.intercept('GET', '**/events/running?token=YOUR_API_TOKEN&tenant=airqo', {
+//       statusCode: 200,
+//       body: {
```

Comment:
@Codebmk, do you still need this code?

Reply:
Yeap I do. It's the tests, they need to be updated. But will do later